### PR TITLE
Add render pass instance renderArea VUs

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: chapters/renderpass.txt [hunk 1]
```diff
@@ -3407,7 +3406,59 @@ pass.
...
+    must: be less than or equal to
+    slink:VkFramebufferCreateInfo::pname:width the pname:framebuffer was
+    created with
+  * [eq]#pname:renderArea.offset.y {plus} pname:renderArea.offset.height#
+    must: be less than or equal to
+    slink:VkFramebufferCreateInfo::pname:height the pname:framebuffer was
+    created with
+endif::VK_VERSION_1_1,VK_KHR_device_group[]
+ifdef::VK_VERSION_1_1,VK_KHR_device_group[]
+  * If pname:pNext chain does not contain
```
Comment (oddhack) [inline chapters/renderpass.txt]: If *the* pNext chain (here and below).

File: chapters/renderpass.txt [hunk 2]
```diff
@@ -3407,7 +3406,59 @@ pass.
   * [[VUID-VkRenderPassBeginInfo-renderPass-00904]]
     pname:renderPass must: be <<renderpass-compatibility,compatible>> with
     the pname:renderPass member of the sname:VkFramebufferCreateInfo
-    structure specified when creating pname:framebuffer.
+    structure specified when creating pname:framebuffer
+ifndef::VK_VERSION_1_1,VK_KHR_device_group[]
+  * pname:renderArea.offset.x must: be greater or equal to 0
```
Comment (gnl21) [inline chapters/renderpass.txt]: "greater *than* or equal to" (and also below).

File: chapters/renderpass.txt (hunk 1 above)
Comment (krOoze) [reply chapters/renderpass.txt]: done

File: chapters/renderpass.txt (hunk 2 above)
Comment (krOoze) [reply chapters/renderpass.txt]: done

**Comment by oddhack** [comment]
This will get merged once conflicts are resolved.

**Comment by krOoze** [comment]
@oddhack rebased

**Pull request merged**